lib/Controller/Schedule.php

```diff
@@ -1011,7 +1025,81 @@ function editForm($eventId)
                 return $element->displayGroupId;
             }, $schedule->displayGroups),
             'help' => $this->getHelp()->link('Schedule', 'Edit'),
-            'reminders' => $scheduleReminders
+            'reminders' => $scheduleReminders,
+            'recurringEvent' => ($schedule->recurrenceType != '') ? true : false,
+            'eventStart' => $eventStart,
+            'eventEnd' => $eventEnd,
+        ]);
+    }
+
+    /**
+     * Shows the Delete a Recurring Event form
+     * @param int $eventId
+     */
+    function deleteRecurrenceForm($eventId)
+    {
+        // Recurring event start/end
+        $eventStart = $this->getSanitizer()->getString('eventStart', 1000);
+        $eventEnd = $this->getSanitizer()->getString('eventEnd', 1000);
+
+        $schedule = $this->scheduleFactory->getById($eventId);
+        $schedule->load();
+
+        if (!$this->isEventEditable($schedule->displayGroups))
```

Comment:
Can we add `{}` to these types of statements, just so we're all doing the same thing going forward